tutorials/server/maas-kvm-pod/maas-kvm-pod.md

```diff
@@ -0,0 +1,204 @@
+---
+id: create-kvm-pods-with-maas
+summary: Learn how to create KVM pods with MAAS. MAAS is an enterprise-grade infrastructure management tool.
+categories: server
+tags: maas, virtualization, pod, kvm, cloud, beginner
+difficulty: 2
+status: Published
+Published: 2017-10-08
+author: Canonical Web Team <webteam@canonical.com>
+
+---
+
+# Create KVM pods with MAAS
+
+## Overview
+Duration: 0:02
+
+[MAAS](https://maas.io/) enables you to treat physical servers like an elastic cloud-like resource. The latest release of MAAS 2.2 introduces “pods” as an operational concept. A MAAS pod effectively describes the availability of resources and enables the creation (or composition) of a machine with a set of those resources. Each pod represents a pool of various available hardware resources, such as CPU, RAM, and (local or remote) storage capacity. A user can allocate the needed resources manually (using the MAAS UI or CLI) or dynamically (using Juju or the MAAS API). That is, machines can be allocated "just in time", based on CPU, RAM, and storage constraints of a specific workload. 
+
+Currently MAAS supports two types of pods, (1) Physical systems with [Intel RSD](https://docs.ubuntu.com/maas/2.2/en/nodes-power-types#bmc-driver-support) and (2) Virtual Machines with KVM (using the virsh interface). Since we want to explore how to better utilize existing hardware, let’s build a test environment with KVM pods.
+
+### Requirements
+
+A testbed environment would require a server with at least 4 CPU cores, 16GB RAM,100GB free disk space, preferably SSD and two NICs. One NIC would be connected to an external network (possibly a DMZ) and the second NIC will be the internal network. MAAS will act as an HTTP proxy and IP gateway between the two networks. MAAS will also provide DNS for all the VMs and servers/pods it will be managing, as well as DHCP. MAAS needs to be installed on only one server/pod and it will be managing all the other pods remotely. MAAS is very versatile. We are focusing here only on one out of many potential KVM pod scenarios.
```

Comment:
I would recommend turning this part into a list, maybe something like this:

### Requirements

A testbed environment would require a server with at least:

* 4 CPU cores
* 16GB RAM
* 100GB free disk space, preferably SSD
* 2 NICs: one connected to an external network (possibly a DMZ) and the second NIC will be the internal network.

MAAS will act as an HTTP proxy and IP gateway between the two networks. MAAS will also provide DNS for all the VMs and servers/pods it will be managing, as well as DHCP. MAAS needs to be installed on only one server/pod and it will be managing all the other pods remotely. MAAS is very versatile. We are focusing here only on one out of many potential KVM pod scenarios.